CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+### citus v9.1.1 (December 18, 2019) ###
+
+* Fixes an issue that caused CTEs to sometimes leak connections
+
+* Fixes counter that keeps track of internal depth in executor
+
+* Fixes a crash when calling a distributed function from PL/pgSQL
+
+* Fixes a crash in IN (.., NULL) queries
+
+* Fixes a bug that caused some noop DML statements to fail
+
+* Fixes a bug that caused inserts into local tables w/ dist. subqueries to crash
+
+* Fixes strange errors in DML with unreachable sublinks
+
+* Fixes a bug that caused column_to_column_name to crash for invalid input
+
+* Fixes a bug causing SQL-executing UDFs to crash when passing in DDL
+
+* Fixes a bug that prevents dropping reference table columns
+
+* Fixes a bug that can cause stmts in SQL func. to run outside of a transaction
```

Comment:
We don't have any abbreviations such as `stmts` or `func.` anywhere in this file.

How does the following sound?
```
Prevents statements in SQL functions to run outside of a transaction
```